base/applications/games/winmine/main.c

```diff
@@ -1092,7 +1092,7 @@ int WINAPI wWinMain( HINSTANCE hInst, HINSTANCE hPrevInst, LPWSTR cmdline, int c
     wc.cbWndExtra = 0;
     wc.hInstance = hInst;
     wc.hIcon = LoadIconW( hInst, MAKEINTRESOURCEW(IDI_WINEMINE) );
-    wc.hCursor = LoadCursorW( 0, (LPWSTR)IDI_APPLICATION );
+    wc.hCursor = LoadCursorW( 0, (LPWSTR)IDC_ARROW );
```

Comment:
Here you are changing to the correct cursor but you forgot to remove the cast?

Reply:
Yep, I did not remove the cast. But as I see, it's used as well in many identical calls too. Is there any reason why the same casts were added there? Maybe I need to remove them too? :thinking: 
For example:
https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/winhlp32/winhelp.c;hb=7e89227a005f3c60ddd1f93dcb797e5583ef857d#l774

Reply:
I think having this cast does not matter so much, so I'd prefer to keep it, same as it's done originally.